.github/workflows/releaser.yml

```diff
@@ -0,0 +1,36 @@
+# .github/workflows/release.yml
+name: goreleaser
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: setup dependencies
+        uses: actions/setup-go@v2
+
+      - name: Login to GHCR
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
```

Comment:
let's use docker hub, i'll help set this up

Reply:
wdyt?

probably docker hub is more common and easier to use, as you don't need to be logged into github (it's more easily accessible)

I've created https://hub.docker.com/r/flashbots/suave-geth and added env vars to the repo settings

Reply:
Did you set `DOCKERHUB_USERNAME` and `DOCKERHUB_TOKEN`?